lms/djangoapps/bulk_email/tasks.py

```diff
@@ -418,7 +418,22 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
         'failed' count above.
     """
```

Comment:
@adampalay I know it will all go into splunk, the bigger question is once it is in splunk will it be easy to pull it all out.